sql_metadb/derived_tables/instance_contributors.sql

```diff
@@ -0,0 +1,57 @@
+--This derived table extracts data for the instance contributors. 
+--It includes the instance uuid, hrid, the contributor's name, if it is primary, type id, type name, type text, name type id and name type.
+--Ordinality has been included.
+DROP TABLE IF EXISTS instance_contributors;
+
+CREATE TABLE instance_contributors AS 
+WITH contribs AS (
+	SELECT 
+		i.id AS instance_id,
+		jsonb_extract_path_text(i.jsonb, 'hrid') AS instance_hrid,
+		jsonb_extract_path_text(ctb.jsonb, 'name') AS contributor_name,
+		jsonb_extract_path_text(ctb.jsonb, 'primary')::boolean AS contributor_is_primary,
+		jsonb_extract_path_text(ctb.jsonb, 'contributorTypeId')::uuid AS contributor_type_id,
+		jsonb_extract_path_text(ctb.jsonb, 'contributorTypeText') AS contributor_type_text,
+		jsonb_extract_path_text(ctb.jsonb, 'contributorNameTypeId')::uuid AS contributor_name_type_id,
+    	ctb.ordinality AS contributor_ordinality
+	FROM 
+		folio_inventory.instance i

```

Comment:
We are using `AS` here even though Postgres does not require it.  I will add this.